src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -199,6 +199,30 @@ public Assembly LoadFromAssemblyName(AssemblyName assemblyName)
             return assembly;
 
         }
+
+        [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
+        [SuppressUnmanagedCodeSecurity]
+        private static extern IntPtr LoadUnmanagedLibraryFromPath(string unmanagedDllPath);
```

Comment:
This may be called InternalLoadUnmanagedDllFromPath based on the naming conventions from https://github.com/kouvel/coreclr/blob/LoadUnmanagedDll/Documentation/botr/mscorlib.md